.github/workflows/pypi.yaml

```diff
@@ -5,28 +5,33 @@ on:
 
 jobs:
   publish:
-    name: Publish to PyPi
+    name: Publish to PyPI
     runs-on: ubuntu-latest
 
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      id-token: write
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3
+          python-version: 3.11
```

Comment:
Good thing I tested on my fork, cause `3` was picking up 3.12 and rez isn't compatible with 3.12, which resulted in an error (since rez is imported from setup.py).